services/security/security.go

```diff
@@ -283,6 +291,14 @@ type CertificateRequest struct {
 type IdentityProperties struct {
 	// State - State
 	Statuses map[string]*string `json:"statuses"`
+	// CloudAgent FQDN
+	CloudFqdn *string `json:"cloudfqdn,omitempty"`
+	// CloudAgent port
+	CloudPort *int32 `json:"cloudport,omitempty"`
+	// CloudAgent authentication port
+	CloudAuthPort *int32 `json:"cloudauthport,omitempty"`
```

Comment:
Ports can be uint . Any specific reason for using signed int?

Reply:
Good point. It's added as int32 to stay consistent with existing definitions for Ports our repo, as well as the azure-sdk-for-go repo.